src/utils/application.ts

```diff
@@ -0,0 +1,75 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { Form, RetrievedApplication, State } from '@/types/application.types';
+
+function formatApplicationState(state: State) {
+  if (!state) return '';
+
+  const s = state.split('/')?.pop() || '';
+  return s.charAt(0).toUpperCase() + s.slice(1);
+}
+
+function isApplicationComplete(application: RetrievedApplication) {
```

Comment:
can you add a unit test for this method? If you want you can have that in a separate PR after this is merged